src/public/version.h

```diff
@@ -0,0 +1,25 @@
+// Copyright 2019 Google Inc. All Rights Reserved.
```

Comment:
I'm skeptical that the data of the copyright should be updated to 2019 unless the code is also being modified instead of just cut-and-pasted.